src/as_execution/abi.rs

```diff
@@ -1120,6 +1599,13 @@ pub fn assembly_script_date_now(mut ctx: FunctionEnvMut<ASEnv>) -> ABIResult<f64
         _ => abi_bail!("failed to get time from interface"),
     };
     let ret = utime as f64;
+    #[cfg(feature = "execution-trace")]
+    ctx.data_mut().trace.push(AbiTrace {
+        name: function_name!().to_string(),
+        params: vec![],
+        return_value: ret.into(),
+        sub_calls: None,
+    });
```

Comment:
Can't comment above but missing ABI data in console.log/info etc. (Maybe it's useless to make them and that's why you didn't made it)

Reply:
console.log/info/warn/error call assembly_script_console which has the traces but maybe we should make different traces?

Reply:
No it's ok